evm_loader/Cargo.lock

```diff
@@ -381,9 +381,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.82"
+version = "1.0.81"
```

Comment:
I see multiple version downgrades in the Cargo.lock file. Could you try to revert the changes and regenerate it with "cargo test" and check if the downgrades are gone?

Reply:
I think the issue is resolved by the last commit.